<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FurnitureDialog</class>
 <widget class="QDialog" name="FurnitureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="chairCheckBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Chair</string>
   </property>
  </widget>
  <widget class="QPushButton" name="furFinishButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QLabel" name="furNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="shelfCheckBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Shelf</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="furLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="tableCheckBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Table</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
